Makefile

```diff
@@ -4,6 +4,7 @@ bindir=/usr/bin
 datadir ?= /usr/share
 sysconfdir ?= /etc
 systemd_unitdir ?= /lib/systemd
+docexamplesdir ?= /usr/share/doc/mender-client/examples/
```

Comment:
In general I think we should stick to one or the other, meaning `/` at the end or not, as it is easy to burn yourself on this, which I think you have done below, when doing `$(dockexamplesdir)/foo`

Reply:
Good catch, thanks.

Reply:
You have a point for the sake of consistency, but keep in mind that `//` is harmless and it is very common in `Makefile` (we have the same situation when `prefix` is empty).

Reply:
It is not harmless, and I don't think that Make automatically trims the double slashes for you, thus say `rm foo/bar` != `rm foo//bar`

Reply:
`rm foo/bar` == `rm foo//bar` in my Terminal